TeensySF/ReadMe.txt

```diff
@@ -0,0 +1,20 @@
+Teensy - a SmoothSwing V2 font by Lord Blako.
```

Comment:
This still doesn't show as a rename, just add/delete, which makes diffing impossible.
Also, where did the "lord blako" part come from?

Reply:
> This still doesn't show as a rename, just add/delete, which makes diffing impossible. 

I'm not sure why. I did it with `git mv` and then edited the newly named file.
https://github.com/profezzorn/SoundFonts/pull/31/commits/ce576eb81106589e9a506a0be144a0ab7d735ff2

> Also, where did the "lord blako" part come from?

I reached out to Khal to clear editing it was ok with him (since it says KR Sabers in the readme). He said no problem, then said Lord Blako actually made it for him. So I contacted Lars as well, and thought it only appropriate to include the creator's name.